client/me/purchases/vat-info/index.tsx

```diff
@@ -84,11 +100,58 @@ export default function VatInfoPage() {
 					</CardHeading>
 					<p className="vat-info__sidebar-paragraph">
 						{ translate(
-							/* translators: %s is the name of taxes in the country (eg: "VAT" or "GST"). */
-							"We currently only provide %(taxName)s invoices to users who are properly registered. %(taxName)s information saved on this page will be applied to all of your account's receipts.",
+							/* translators: %s is the name of taxes in the country (eg: "VAT" or "GST") or a generic fallback string of tax names */
+							'The %(taxName)s details saved on this page will be applied to all receipts in your account.',
 							{
-								textOnly: true,
-								args: { taxName: taxName ?? translate( 'VAT', { textOnly: true } ) },
+								args: { taxName: taxName ?? translate( 'tax (VAT/GST/CT)', { textOnly: true } ) },
+							}
+						) }
+						<br />
+						<br />
+						{ translate(
+							/* translators: %s is the name of taxes in the country (eg: "VAT" or "GST") or a generic fallback string of tax names */
```

Comment:
I'll be curious to see if this works or if it reports that `taxName` has no description. My understanding is that `translators` comments need to be on the line immediately preceding the string with the placeholder, but in this case there's two differences and I don't know how they will affect this: there are two `translators` comments one after another, and the string is broken up into multiple strings. 🤷  It might be fine.

Reply:
I would put this comment directly above the line where `%s` is used (actually it looks like `%s` is never used in this translation? maybe you meant `%(taxName)s`?).

Reply:
Done! Thanks!